src/main/scala/cromwell/util/Encryption.scala

```diff
@@ -23,6 +23,22 @@ case object Aes256Cbc {
 
   val ranGen = new SecureRandom()
 
+  lazy val isAvailable = {
+    try {
+      // TODO: Easier way to test if using this key size is supported?
+      val key = new Array[Byte](keySize)
+      val iv = new Array[Byte](blockSize / 8)
+      ranGen.nextBytes(key)
+      ranGen.nextBytes(iv)
+      init(Cipher.ENCRYPT_MODE, key, iv)
+      init(Cipher.DECRYPT_MODE, key, iv)
+      true
+    } catch {
+      case e: InvalidKeyException =>
+        false
+    }
+  }
```

Comment:
I didn't like my first proof of concept, so I just refactored the check into a pure test utility.

Regarding production code, I'm fine with whatever, but at the moment I'm aiming for a working build to unblock my two PRs. :smile:

There are other devops issues already. For example: I have no idea how one installs JCE unlimited using continuous integration or automation. But, what I don't know could fill a warehouse. I wasn't aware of how to even script installation of Oracle Java until I saw someone else do that too.